app/redux/Store.js

```diff
@@ -52,6 +53,7 @@ export default initializeStore = () => {
             'history',
             'killSwitch',
             'suggestions',
+            'appState',
         ],
         // note, everything in sets is to be persisted so not blacklisting or transforming them
         transforms: [
```

Comment:
remove workout blacklist and instead create a filter to store the remvoe and restore count

Reply:
Do not understand.

Reply:
Done